### ISSUE TITLE
Manual assignment or supervisors to groups

### ISSUE THREAD

**Issue by phish108** [BLC-HTWChur/moodle-mod_groupselect#15]
Currently, only automatic supervisor assignment is possible. It should be possible to assign specific supervisors manually.

**Comment by SamHAnderson** [comment]
Agreed, this setting as developed is totally confusing to our instructors (and those they would designate as supervisors) due to the random nature of it. We ended up hiding the setting with CSS. Manual assignment would be preferred, or at least an admin toggle to disable the option entirely.
